.github/workflows/pr-review-check.yml

```
@@ -0,0 +1,37 @@
+name: "PR Review check"
+
+env:
+  TRIGGER_LABELS: '["CI", "preview", "skip-tests"]'
+  DEPENDABOT_CREATOR: 'dependabot[bot]'
+  MILESTONE_NAME: 'Validating'
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+  pull_request:
+    branches:
+      - develop
+    types:
+      - labeled
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger validate on approval
```

```suggestion
      - name: Trigger validation on approval
```

To stay consistent with L#31